### PR TITLE
New Support System - Kill Spoffys sanity once and for all (sorry)

### PR DESCRIPTION
**Pull request by wurzel0701**
## What type of PR is this.
1. [X] Bug
2. [X] Change
3. [X] Enhancement

### What have you changed and why?
Information:
This is the PR for the entire support, including alot of currently unwanted (unfair) supports. For detailed states of the project check out (insert link for my fork repo).

How the system works:

(Currently a non detail version, as this PR woll only be a draft for now)
AI units can call for support in attackDrill.sqf
AI units can call for support in their killed EH
Several other events trigger supports too (Missile fired, damaged)

The chooseSupport.sqf chooses a list of suited supports against the given target (AA vs air vehicles, AT vs vehicles, general supports against infantry)
Then sendSupport.sqf uses the supportAvailable.sqf to check if the given support is available
Each support has a <SupportName>Available.sqf file, which defines the conditions for availability, which gets called by supportAvailable.sqf

(Will continue later)
    
### Please specify which Issue this PR Resolves.
closes #XXXX

### Please verify the following and ensure all checks are completed.

1. [X] Have you loaded the mission in singleplayer?
2. [X] Have you loaded the mission in LAN host?
3. [X] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [X] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
Play an entire campaign
********************************************************
Notes:

### CODE REVIEW

**Comment by Bob-Murphy** [comment]
The title is correct and incorrect at the same time. Please add more information :D

File: A3-Antistasi/Templates/Vanilla_Inv_CSAT_Altis.sqf [hunk 1]
```diff
@@ -78,7 +78,7 @@ groupsCSATSquad =
 	[
 	CSATSquad,
 	["O_Soldier_SL_F","O_Soldier_AR_F","O_Soldier_GL_F","O_soldier_M_F","O_Soldier_AT_F","O_Soldier_AAT_F","O_Soldier_A_F","O_medic_F"],
-	["O_Soldier_SL_F","O_Soldier_LAT_F","O_Soldier_TL_F","O_Soldier_AR_F","O_Soldier_A_F","O_support_Mort_F","O_support_AMort_F","O_medic_F"],
+	//["O_Soldier_SL_F","O_Soldier_LAT_F","O_Soldier_TL_F","O_Soldier_AR_F","O_Soldier_A_F","O_support_Mort_F","O_support_AMort_F","O_medic_F"],
```
Comment (Lazejun) [inline A3-Antistasi/Templates/Vanilla_Inv_CSAT_Altis.sqf:81]: Removed with #1420 Please use the Latest version if you change Templates

**Comment by jaj22** [comment]
Still has masses of duplicated description.ext code that should be pulled into a MissionDescription include. Not picky about how this is handled but it should be.

**Comment by CalebSerafin** [comment]
> Still has masses of duplicated description.ext code that should be pulled into a MissionDescription include. Not picky about how this is handled but it should be.

See https://github.com/official-antistasi-community/A3-Antistasi/pull/1637

**Comment by CalebSerafin** [comment]
* [ ] Indentation & Headers
* [ ] Naming Conventions & Grammar
* [ ] Macros
* [ ] Param & Call
* [ ] Global Variables
* [ ] Local Variables
* [ ] Logic

**Review by Bob-Murphy — APPROVED** [review]
merge without proper review as discussed

**Pull request merged**